desktop-widgets/tab-widgets/TabDiveInformation.cpp

```diff
@@ -295,8 +291,8 @@ void TabDiveInformation::on_waterTypeCombo_activated(int index)
 	else
 		ui->salinityText->clear();
 	divesEdited(Command::editWaterTypeUser(combobox_salinity, false));
-	// If salinity differs from that downloaded from dc, show warning
-	if (manualDive || dc_salinity == combobox_salinity)
+	// If salinity differs from that downloaded from dc (if it has one), show warning
```

Comment:
This comment is now incorrect - we are not comparing the salinity to something that was downloaded from a dive computer, but to a calculated average.

Reply:
Done